tests/dto/thirdparty.test.ts

```diff
@@ -561,6 +561,29 @@ describe('thirdparty', () => {
     expect(transactionRequestsPostRequest).toBeDefined()
   })
 
+  test('ThirdpartyRequestsVerificationsIDPutResponse', () => {
+    const thirdpartyRequestsVerificationsIdPutResponse: Schemas.ThirdpartyRequestsVerificationsIDPutResponse = {
+      authenticationResponse: 'VERIFIED'
+    }
+    expect(thirdpartyRequestsVerificationsIdPutResponse).toBeDefined()
+  })
+
+  test('ThirdpartyRequestsVerificationsPostRequest', () => {
+    const thirdPartyRequestsVerificationsPostRequest: Schemas.ThirdpartyRequestsVerificationsPostRequest = {
+      verificationRequestId: correlationId,
+      challenge: 'some challenge base64 encoded',
+      value: {
+        authenticationInfo: {
+          authentication: 'U2F',
```

Comment:
I think this request body needs to be updated to handle `clientDataJson`/`authenticatorData`/`signature`/`userHandle` https://webauthn.guide/#authentication

Reply:
Not sure how we want to change this payload to include `FIDOPublicKeyCredentialAssertion` but lets do that in another PR.